zerver/lib/narrow.py

```diff
@@ -273,14 +273,18 @@ def __init__(
             "near": self.by_near,
             "id": self.by_id,
             "search": self.by_search,
-            "pm-with": self.by_pm_with,
+            "dm": self.by_dm,
+            # "pm-with" is a deprecated operator.
```

Comment:
Again, I think saying "pm-with: is a deprecated alias of dm:" would make this a lot easier to follow for someone who doesn't already know how this works.

Reply:
Done.